src/python/pants/base/exception_sink.py

```diff
@@ -212,6 +213,23 @@ def reset_log_location(cls, new_log_location: str) -> None:
         cls._pid_specific_error_fileobj = pid_specific_error_stream
         cls._shared_error_fileobj = shared_error_stream
 
+    @classmethod
+    def open_pid_specific_error_stream(cls, path):
+        try:
+            ret = safe_open(path, mode="w")
+        except Exception:
```

Comment:
This is just redundant IIUC and the whole try / except can be removed.

Reply:
Yeah that's a 🤦‍♂️